lmfit/printfuncs.py

```diff
@@ -75,7 +75,7 @@ def gformat(val, length=11):
         if expon > 0:
             prec -= expon
     fmt = '{0: %i.%i%s}' % (length, prec, form)
-    return fmt.format(val)
+    return fmt.format(val)[:length]
```

Comment:
I am not sure if this is correct change... which problem does it solve? In line 66 the length is set to ```max(length, 7)``` now with chopping the return value to ```[:length]``` this could give weird results as shown below:

Let's take ```val = 1.234567890```; as long as length => 7 the old and new ```gformat``` will give the same result, but for length < 7 the behavior changes:

"old" gformat(val, length=5) -->  '  1e+00'
"new" gformat(val, length=5) -->  '  1e+'

is that intentional?

Reply:
Sorry, should have raised an issue about this, but it seemed like a small buglet.  I'll describe in more detail below.

Reply:
there was indeed no need to raise an issue about this, I clearly misread the proposed code change (see below)